server/sonar-web/src/main/js/app/components/nav/component/ComponentNavBranch.tsx

```diff
@@ -187,27 +202,33 @@ export default class ComponentNavBranch extends React.PureComponent<Props, State
   );
 
   render() {
-    const { branches, currentBranch } = this.props;
+    const { branchLikes, currentBranchLike } = this.props;
 
     if (this.context.onSonarCloud && !this.context.branchesEnabled) {
       return null;
     }
 
+    const displayName = getBranchLikeDisplayName(currentBranchLike);
+
     if (!this.context.branchesEnabled) {
       return (
         <div className="navbar-context-branches">
-          <BranchIcon branch={currentBranch} className="little-spacer-right" fill={theme.gray80} />
-          <span className="note">{currentBranch.name}</span>
+          <BranchIcon
+            branchLike={currentBranchLike}
+            className="little-spacer-right"
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([typescript:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3010&issues=AWIFEiOQJ6stENANvNAP&open=AWIFEiOQJ6stENANvNAP)

[ISSUE_KEY]: AWIFEiOQJ6stENANvNAP